server/services/database/documents.js

```diff
@@ -4,22 +4,31 @@ const pool = new Pool(config);
 
 const query =
   "select\
-          d.id, d.format, d.name,\
-          d.status, d.submission_date, d.due_date,\
-          lf.name as from_language_name,\
-          lt.name as to_language_name,\
-          u.name as owner_name\
-      from documents as d\
-      inner join languages as lf\
-      on  lf.code = d.from_language_code\
-      inner join languages as lt\
-      on lt.code = d.to_language_code\
-      inner join users as u\
-      on u.id = d.owner_id";
+  d.id, d.format, d.name,\
+    d.status, d.submission_date, d.due_date,\
+    lf.name as from_language_name,\
+    lt.name as to_language_name,\
+    u.name as owner_name,\
+    trnsl.start_date, trnsl.translator_name\
+  from documents as d\
+  inner join languages as lf\
+  on  lf.code = d.from_language_code\
+  inner join languages as lt\
+  on lt.code = d.to_language_code\
+  inner join users as u\
+  on u.id = d.owner_id\
+  left join(\
+    select\
+           t.document_id,\
+    t.start_date,\
+    tu.name as translator_name\
+    from translations as t\
+    inner join users as tu\
+        on tu.id = t.user_id\
+  ) trnsl on trnsl.document_id = d.id";
```

Comment:
I think this can be simpler. In general is better to avoid using nested SQL query as it's more complex to read and maintain. Try something like:
```
  left join translations t on t.document_id = d.id 
  left join users u2 on u2.id = t.user_id
```